### PR TITLE
qf-kvm-page: set next_start_key correctly & some fixes

### PR DESCRIPTION
**Pull request by icehess**

### CODE REVIEW

File: applications/crossbar/src/modules_v1/cb_vmboxes_v1.erl [hunk 1]
```diff
@@ -30,6 +30,7 @@
 -include_lib("kazoo_documents/include/kazoo_documents.hrl").
 
 -define(CB_LIST, <<"vmboxes/crossbar_listing">>).
+-define(MSG_LISITNG_BY_MAILBOX, <<"mailbox_messages/listing_by_mailbox">>).
```
Comment (lazedo) [inline applications/crossbar/src/modules_v1/cb_vmboxes_v1.erl]: @icehess `MSG_LISITNG_BY_MAILBOX` misspelled (4 places).

**Comment by icehess** [comment]
please do not merge this until refactoring voicemail is done.

**Comment by fenollp** [comment]
Any chances https://github.com/2600hz/kazoo/pull/2570 helps here?

**Comment by icehess** [comment]
@fenollp I'm looking to it to see if it can help with this, but anyway I closed this pr since the #2571 is fixing this issue.

**Pull request closed without merging**